realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -34,6 +34,7 @@
 public class RealmQuery<E extends RealmObject> {
 
     private Realm realm;
+    private Table tableSpec;
```

Comment:
Please don't name it `tableSpec` as we have an internal class called `TableSpec`.